Sources/ArcGISToolkit/Components/FeatureTemplatePicker/FeatureTemplatePicker.swift

```diff
@@ -0,0 +1,340 @@
+                            ContentUnavailableView(
+                                String.noFeatureTemplatesFoundTitle,
+                                systemImage: "magnifyingglass.circle",
+                                description: Text(String.noFeatureTemplatesFoundDetail)
+                            )
+                        } else {
+                            // Fallback on earlier versions
+                            Text(String.noFeatureTemplatesFoundDetail)
+                        }
+                    }
+                }
+            }
+        }
+        .task {
+            await model.generateFeatureTemplates()
+        }
+    }
+}
+
+private extension String {
+    static var noFeatureTemplatesTitle: String {
+        String(
+            localized: "No Feature Templates",
+            bundle: .toolkitModule,
+            comment: """
+                 A title for showing a view that tells the user there are no feature templates.
+                 """
+        )
+    }
+    static var noFeatureTemplatesDetail: String {
+        String(
+            localized: "There are no feature templates available.",
+            bundle: .toolkitModule,
+            comment: """
+                 Details for showing a view that tells the user there are no feature templates
+                 available.
+                 """
+        )
+    }
+    static var searchTemplatesPrompt: String {
+        String(
+            localized: "Search templates",
+            bundle: .toolkitModule,
+            comment: """
+                 A prompt in the search templates text box that instructs the user that they
+                 can type in the field to search for templates.
+                 """
+        )
+    }
+    static var noFeatureTemplatesFoundTitle: String {
+        String(
+            localized: "Nothing Found",
+            bundle: .toolkitModule,
+            comment: """
+                 A title for showing a view that tells the user there were no feature templates
+                 found that match their search criteria.
+                 """
+        )
+    }
+    static var noFeatureTemplatesFoundDetail: String {
+        String(
+            localized: "There were no feature templates found that match the search criteria.",
+            bundle: .toolkitModule,
+            comment: """
+                 Details for showing a view that tells the user there were no feature templates
+                 found that match their search criteria.
+                 """
+        )
+    }
+}
+
+/// View of a feature template.
+private struct FeatureTemplateView: View {
+    let info: FeatureTemplateInfo
+    @Binding var selection: FeatureTemplateInfo?
+    
+    var body: some View {
+        HStack {
+            Label {
+                Text(info.template.name)
+                    .lineLimit(1)
+            } icon: {
+                if let image = info.image {
+                    Image(uiImage: image)
```

Comment:
Big symbol images can get unwieldy.

```suggestion
                    Image(uiImage: image)
                        .resizable()
                        .aspectRatio(contentMode: .fit)
```
Before
![image](https://github.com/user-attachments/assets/81791f57-75e5-446d-af45-f09c02f2fff8)

After:
![image](https://github.com/user-attachments/assets/f63ff236-590d-4e4e-b052-457f066db66c)

And it still works with "regular" symbols, including showing size gradation:

![image](https://github.com/user-attachments/assets/bf47c95f-9216-4951-9290-37f7cac6f8da)

Reply:
Nice one!